devel -> staging

pmattmann opened the pull request:
- Admin: Checklist
- Checklist-Overview
- Print: Activity-Overview for Courses

renovate[bot] committed:
Merge pull request #6143 from ecamp/renovate/rector-rector-1.x

chore(deps): update dependency rector/rector to v1.2.7

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.26

renovate[bot] committed:
Merge pull request #6144 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.26

BacLuc committed:
Merge pull request #6105 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.16.11

pmattmann committed:
use new parameter-names

pmattmann committed:
dashboard add scroll-to-today

manuelmeister committed:
Adjust design

BacLuc committed:
Merge pull request #6134 from ecamp/renovate/thollander-actions-comment-pull-request-3.x

chore(deps): update thollander/actions-comment-pull-request action to v3

renovate[bot] committed:
chore(deps): update dependency vite to v5.4.9

renovate[bot] committed:
chore(deps): update dependency @nuxt/eslint to v0.6.0

renovate[bot] committed:
chore(deps): update dependency @nuxt/eslint-config to v0.6.0

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.6

renovate[bot] committed:
Merge pull request #6146 from ecamp/renovate/vite-5.x

chore(deps): update dependency vite to v5.4.9

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.9.0

renovate[bot] committed:
Merge pull request #6148 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.6

renovate[bot] committed:
Merge pull request #6149 from ecamp/renovate/nuxt-eslint-0.x

chore(deps): update dependency @nuxt/eslint to v0.6.0

renovate[bot] committed:
chore(deps): update dependency @nuxt/eslint-config to v0.6.0

renovate[bot] committed:
Merge pull request #6150 from ecamp/renovate/nuxt-eslint-config-0.x

chore(deps): update dependency @nuxt/eslint-config to v0.6.0

renovate[bot] committed:
Merge pull request #6151 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.9.0

renovate[bot] committed:
chore(deps): update vitest monorepo to v2.1.3

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6147 from ecamp/renovate/vitest-monorepo

chore(deps): update vitest monorepo to v2.1.3

renovate[bot] committed:
Merge pull request #6152 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): lock file maintenance

manuelmeister committed:
Merge pull request #6115 from BacLuc/fix-reverse-proxy

reverse-proxy-nginx.conf: reverse proxy upgrade response for websockets

manuelmeister committed:
Change copy nodes text icon

manuelmeister committed:
Filter items by checklist

pmattmann committed:
Scroll to today

pmattmann committed:
Allow delete ChecklistItem

manuelmeister committed:
Separate checklist overviews

pmattmann committed:
improve data-loading

pmattmann committed:
Merge pull request #6026 from pmattmann/feature/checklist-overview-draft

Checklist-Overview with Activities

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.7

renovate[bot] committed:
Merge pull request #6153 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.56.0

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6155 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.7

renovate[bot] committed:
Merge pull request #6156 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.56.0

renovate[bot] committed:
Merge pull request #6157 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6158 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6159 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
fix(deps): update dependency puppeteer-core to v23.6.0

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.27

renovate[bot] committed:
Merge pull request #6161 from ecamp/renovate/puppeteer-core-23.x

fix(deps): update dependency puppeteer-core to v23.6.0

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.8

renovate[bot] committed:
chore(deps): update dependency @testing-library/jest-dom to v6.6.1

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6162 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.27

renovate[bot] committed:
Merge pull request #6160 from ecamp/renovate/testing-library-monorepo

chore(deps): update dependency @testing-library/jest-dom to v6.6.1

renovate[bot] committed:
Merge pull request #6164 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.8

renovate[bot] committed:
Merge pull request #6165 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.16.12

renovate[bot] committed:
chore(deps): update dependency node to v22.10.0

renovate[bot] committed:
Merge pull request #6166 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.16.12

renovate[bot] committed:
Merge pull request #6167 from ecamp/renovate/node-22.x

chore(deps): update dependency node to v22.10.0

usu committed:
Merge pull request #6154 from pmattmann/feature/delete-checklist-item

Delete ChecklistItem

usu committed:
improve deleting UX

renovate[bot] committed:
chore(deps): update dependency @nuxtjs/tailwindcss to v6.12.2

renovate[bot] committed:
chore(deps): update dependency @testing-library/jest-dom to v6.6.2

renovate[bot] committed:
Merge pull request #6168 from ecamp/renovate/nuxtjs-tailwindcss-6.x

chore(deps): update dependency @nuxtjs/tailwindcss to v6.12.2

renovate[bot] committed:
Merge pull request #6169 from ecamp/renovate/testing-library-monorepo

chore(deps): update dependency @testing-library/jest-dom to v6.6.2

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.10.0

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.56.1

renovate[bot] committed:
Merge pull request #6172 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.10.0

renovate[bot] committed:
Merge pull request #6175 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.56.1

renovate[bot] committed:
chore(deps): update pulumi/pulumi-nodejs docker tag to v3.137.0

renovate[bot] committed:
fix(deps): update dependency @pulumi/pulumi to v3.137.0

renovate[bot] committed:
Merge pull request #6177 from ecamp/renovate/pulumi-pulumi-nodejs-3.x

chore(deps): update pulumi/pulumi-nodejs docker tag to v3.137.0

renovate[bot] committed:
Merge pull request #6178 from ecamp/renovate/pulumi-pulumi-3.x

fix(deps): update dependency @pulumi/pulumi to v3.137.0

renovate[bot] committed:
chore(deps): update dependency @sentry/vite-plugin to v2.22.6

renovate[bot] committed:
chore(deps): update dependency phpstan/phpstan to v1.12.7

renovate[bot] committed:
Merge pull request #6179 from ecamp/renovate/sentry-vite-plugin-2.x

chore(deps): update dependency @sentry/vite-plugin to v2.22.6

renovate[bot] committed:
Merge pull request #6180 from ecamp/renovate/phpstan-packages

chore(deps): update dependency phpstan/phpstan to v1.12.7

renovate[bot] committed:
chore(deps): update dependency @testing-library/jest-dom to v6.6.2

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.14.3

renovate[bot] committed:
Merge pull request #6171 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.14.3

renovate[bot] committed:
chore(deps): update dependency rector/rector to v1.2.8

renovate[bot] committed:
Merge pull request #6182 from ecamp/renovate/rector-rector-1.x

chore(deps): update dependency rector/rector to v1.2.8

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.10

renovate[bot] committed:
chore(deps): update dependency @eslint/compat to v1.2.1

renovate[bot] committed:
chore(deps): update eslint monorepo to v9.13.0

renovate[bot] committed:
Merge pull request #6173 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.10

renovate[bot] committed:
Merge pull request #6183 from ecamp/renovate/eslint-compat-1.x

chore(deps): update dependency @eslint/compat to v1.2.1

renovate[bot] committed:
Merge pull request #6184 from ecamp/renovate/eslint-monorepo

chore(deps): update eslint monorepo to v9.13.0

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.16.13

renovate[bot] committed:
Merge pull request #6185 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.16.13

renovate[bot] committed:
fix(deps): update dependency api-platform/core to v3.4.4

pmattmann committed:
Merge pull request #6174 from ecamp/renovate/testing-library-monorepo

chore(deps): update dependency @testing-library/jest-dom to v6.6.2

renovate[bot] committed:
chore(deps): update dependency @sentry/vite-plugin to v2.22.6

pmattmann committed:
Merge pull request #6046 from simfeld/feature/edit-delete-checklists

feat: edit and delete checklists

pmattmann committed:
Merge pull request #6181 from ecamp/renovate/sentry-vite-plugin-2.x

chore(deps): update dependency @sentry/vite-plugin to v2.22.6

pmattmann committed:
fix breaking change api-platform 3.4

BacLuc commented:
Its not very cool with the many console errors.
I would vote to either solve the problem or move the button into the overview (temporarily)

But it seems we only get one sentry error for it:
https://ecamp.sentry.io/share/issue/9336a486894c45969fc8f650d1570aef/

maybe we don't have to solve the overall problem.

BacLuc committed:
frontend: set vitest test workers to 1

That makes our ci more stable by eliminating the
"jest is not defined" errors.
It also increased the duration from 1 minutes to 2.5 minutes.

BacLuc committed:
continuous-integration.yml: do not run frontend checks if not needed

renovate[bot] committed:
chore(deps): update dependency eslint-plugin-vue to v9.29.1

renovate[bot] committed:
Merge pull request #6189 from ecamp/renovate/eslint-plugin-vue-9.x

chore(deps): update dependency eslint-plugin-vue to v9.29.1

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.28

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.11.0

renovate[bot] committed:
Merge pull request #6192 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.28

pmattmann commented:
> Its not very cool with the many console errors. I would vote to either solve the problem or move the button into the overview (temporarily)
> 
> But it seems we only get one sentry error for it: https://ecamp.sentry.io/share/issue/9336a486894c45969fc8f650d1570aef/
> 
> maybe we don't have to solve the overall problem.

The use case 'delete checklist' is probably not used very often. 
So I could live with it if we don't let ourselves be blocked - but deliver the adaptation quickly.

pmattmann committed:
set exception_to_status

renovate[bot] committed:
Merge pull request #6193 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.11.0

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.16.14

renovate[bot] committed:
Merge pull request #6197 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.16.14

renovate[bot] committed:
fix(deps): update tiptap to v2.9.0

renovate[bot] committed:
chore(deps): update actions/cache digest to 6849a64

renovate[bot] committed:
chore(deps): update dependency @babel/eslint-parser to v7.25.9

renovate[bot] committed:
Merge pull request #6199 from ecamp/renovate/tiptap

fix(deps): update tiptap to v2.9.0

renovate[bot] committed:
chore(deps): update helm release kubernetes-dashboard to v7.9.0

pmattmann committed:
Merge pull request #6200 from ecamp/renovate/babel-monorepo

chore(deps): update dependency @babel/eslint-parser to v7.25.9

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.12

renovate[bot] committed:
Merge pull request #6194 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.12

renovate[bot] committed:
Merge pull request #6202 from ecamp/renovate/kubernetes-dashboard-7.x

chore(deps): update helm release kubernetes-dashboard to v7.9.0

renovate[bot] committed:
chore(deps): update helm release oauth2-proxy to v7.7.28

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.16.15

renovate[bot] committed:
chore(deps): update dependency vite to v5.4.10

renovate[bot] committed:
Merge pull request #6203 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.16.15

renovate[bot] committed:
Merge pull request #6204 from ecamp/renovate/vite-5.x

chore(deps): update dependency vite to v5.4.10

renovate[bot] committed:
fix(deps): update tiptap to v2.9.1

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.15.0

renovate[bot] committed:
Merge pull request #6205 from ecamp/renovate/tiptap

fix(deps): update tiptap to v2.9.1

renovate[bot] committed:
Merge pull request #6206 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.15.0

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.35.0

renovate[bot] committed:
Merge pull request #6190 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.35.0

pmattmann committed:
Merge pull request #6195 from ecamp/renovate/oauth2-proxy-7.x

chore(deps): update helm release oauth2-proxy to v7.7.28

renovate[bot] committed:
chore(deps): update actions/checkout digest to 11bd719

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.17.0

renovate[bot] committed:
Merge pull request #6208 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.17.0

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.13

renovate[bot] committed:
Merge pull request #6209 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.13

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.14

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.17.1

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.16.1

renovate[bot] committed:
Merge pull request #6213 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.17.1

renovate[bot] committed:
Merge pull request #6210 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.16.1

renovate[bot] committed:
Merge pull request #6211 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.14

pmattmann committed:
Fix CI-Badge

pmattmann committed:
Add pg-admin

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.15

renovate[bot] committed:
Merge pull request #6216 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.15

renovate[bot] committed:
chore(deps): update php

BacLuc committed:
Merge pull request #6201 from ecamp/renovate/php

chore(deps): update php

BacLuc committed:
Merge pull request #6207 from ecamp/renovate/actions-checkout-digest

chore(deps): update actions/checkout digest to 11bd719

BacLuc committed:
Merge pull request #6214 from ecamp/feature/readme-fix-ci-badge

Fix CI-Badge

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.17.0

renovate[bot] committed:
Merge pull request #6217 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.17.0

pmattmann committed:
Merge pull request #6198 from ecamp/renovate/actions-cache-digest

chore(deps): update actions/cache digest to 6849a64

pmattmann committed:
Fix Translation-Badges

shields.io has deactivated the JSONPath array filter. 
Query string of Crowdin badges must therefore now contain fixed position numbers.

See here:
https://shields.io/blog/GHSA-rxvx-x284-4445#2-users-of-dynamic-jsontomlyaml-badges

old:
```
$.progress[?(@.data.languageId=='de-CH')].data.translationProgress
```
new:
```
$.progress.0.data.translationProgress
```

pmattmann committed:
Merge pull request #6219 from ecamp/bugfix/readme-translation-progress

README: fix Translation-Badges

renovate[bot] committed:
fix(deps): update symfony packages to v7.1.6

pmattmann committed:
Merge pull request #6215 from pmattmann/feature/pg-admin

Add pg-admin

renovate[bot] committed:
chore(deps): update dependency @nuxt/eslint to v0.6.1

renovate[bot] committed:
Merge pull request #6225 from ecamp/renovate/nuxt-eslint-0.x

chore(deps): update dependency @nuxt/eslint to v0.6.1

renovate[bot] committed:
chore(deps): update dependency @nuxt/eslint-config to v0.6.1

renovate[bot] committed:
Merge pull request #6226 from ecamp/renovate/nuxt-eslint-config-0.x

chore(deps): update dependency @nuxt/eslint-config to v0.6.1

renovate[bot] committed:
chore(deps): update vitest monorepo to v2.1.4

renovate[bot] committed:
fix(deps): update dependency puppeteer-core to v23.6.1

renovate[bot] committed:
chore(deps): update dependency @eslint/compat to v1.2.2

renovate[bot] committed:
Merge pull request #6228 from ecamp/renovate/puppeteer-core-23.x

fix(deps): update dependency puppeteer-core to v23.6.1

renovate[bot] committed:
Merge pull request #6227 from ecamp/renovate/vitest-monorepo

chore(deps): update vitest monorepo to v2.1.4

renovate[bot] committed:
Merge pull request #6229 from ecamp/renovate/eslint-compat-1.x

chore(deps): update dependency @eslint/compat to v1.2.2

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.12.0

renovate[bot] committed:
chore(deps): update dependency eslint-plugin-vue to v9.30.0

renovate[bot] committed:
Merge pull request #6223 from ecamp/renovate/symfony

fix(deps): update symfony packages to v7.1.6

renovate[bot] committed:
Merge pull request #6230 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.12.0

renovate[bot] committed:
Merge pull request #6231 from ecamp/renovate/eslint-plugin-vue-9.x

chore(deps): update dependency eslint-plugin-vue to v9.30.0

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.16

renovate[bot] committed:
chore(deps): update dependency @types/node to v20.17.2

renovate[bot] committed:
Merge pull request #6233 from ecamp/renovate/node-20.x

chore(deps): update dependency @types/node to v20.17.2

renovate[bot] committed:
chore(deps): update node.js to v22.10.0

renovate[bot] committed:
Merge pull request #6232 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.16

renovate[bot] committed:
Merge pull request #6176 from ecamp/renovate/node-22.x

chore(deps): update node.js to v22.10.0

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.12.2

renovate[bot] committed:
Merge pull request #6234 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.12.2

renovate[bot] committed:
chore(deps): update dependency node to v22.11.0

renovate[bot] committed:
Merge pull request #6235 from ecamp/renovate/node-22.x

chore(deps): update dependency node to v22.11.0

renovate[bot] committed:
chore(deps): lock file maintenance

BacLuc committed:
api: temporary downgrade symfony/framework-bundle to 7.1.5

Because 7.1.6 conflicts with api-platform/core 3.4.4

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.17

BacLuc committed:
Merge pull request #5948 from ecamp/renovate/api-platform-core-3.x

fix(deps): update dependency api-platform/core to v3.4.4

BacLuc committed:
frontend: update snapshot of ETextArea.spec.js

renovate[bot] committed:
Merge pull request #6236 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.17

renovate[bot] committed:
fix(deps): update dependency api-platform/core to v3.4.5

renovate[bot] committed:
Merge pull request #6196 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6239 from ecamp/renovate/api-platform-core-3.x

fix(deps): update dependency api-platform/core to v3.4.5

renovate[bot] committed:
Merge pull request #6240 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): update node.js to v22.11.0

renovate[bot] committed:
fix(deps): update dependency @pulumi/aws to v6.57.0

renovate[bot] committed:
Merge pull request #6241 from ecamp/renovate/node-22.x

chore(deps): update node.js to v22.11.0

renovate[bot] committed:
Merge pull request #6242 from ecamp/renovate/pulumi-aws-6.x

fix(deps): update dependency @pulumi/aws to v6.57.0

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): update dependency eslint-plugin-n to v17.12.0

renovate[bot] committed:
chore(deps): update dependency eslint-plugin-cypress to v4.1.0

renovate[bot] committed:
Merge pull request #6244 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6245 from ecamp/renovate/eslint-plugin-n-17.x

chore(deps): update dependency eslint-plugin-n to v17.12.0

renovate[bot] committed:
Merge pull request #6246 from ecamp/renovate/eslint-plugin-cypress-4.x

chore(deps): update dependency eslint-plugin-cypress to v4.1.0

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6247 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.18.18

renovate[bot] committed:
chore(deps): lock file maintenance

renovate[bot] committed:
Merge pull request #6248 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.18.18

renovate[bot] committed:
Merge pull request #6249 from ecamp/renovate/lock-file-maintenance

chore(deps): lock file maintenance

renovate[bot] committed:
chore(deps): update docker/dockerfile docker tag to v1.11

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.36.0

renovate[bot] committed:
Merge pull request #6250 from ecamp/renovate/docker-dockerfile-1.x

chore(deps): update docker/dockerfile docker tag to v1.11

renovate[bot] committed:
fix(deps): update dependency webonyx/graphql-php to v15.18.0

renovate[bot] committed:
chore(deps): update dependency @testing-library/jest-dom to v6.6.3

renovate[bot] committed:
chore(deps): update helm release kubernetes-dashboard to v7.10.0

renovate[bot] committed:
Merge pull request #6252 from ecamp/renovate/webonyx-graphql-php-15.x

fix(deps): update dependency webonyx/graphql-php to v15.18.0

renovate[bot] committed:
Merge pull request #6253 from ecamp/renovate/kubernetes-dashboard-7.x

chore(deps): update helm release kubernetes-dashboard to v7.10.0

renovate[bot] committed:
Merge pull request #6251 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.36.0

renovate[bot] committed:
Merge pull request #6254 from ecamp/renovate/testing-library-monorepo

chore(deps): update dependency @testing-library/jest-dom to v6.6.3

renovate[bot] committed:
chore(deps): update docker/dockerfile docker tag to v1.11

renovate[bot] committed:
chore(deps): update node.js to v22.11.0

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.19.0

renovate[bot] committed:
Merge pull request #6257 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.19.0

pmattmann committed:
Merge pull request #6243 from ecamp/renovate/node-22.x

chore(deps): update node.js to v22.11.0

pmattmann committed:
Merge pull request #6255 from ecamp/renovate/docker-dockerfile-1.x

chore(deps): update docker/dockerfile docker tag to v1.11

renovate[bot] committed:
chore(deps): update amazon/aws-cli docker tag to v2.19.1

renovate[bot] committed:
chore(deps): update eslint monorepo to v9.14.0

renovate[bot] committed:
Merge pull request #6258 from ecamp/renovate/amazon-aws-cli-2.x

chore(deps): update amazon/aws-cli docker tag to v2.19.1

renovate[bot] committed:
Merge pull request #6259 from ecamp/renovate/eslint-monorepo

chore(deps): update eslint monorepo to v9.14.0

renovate[bot] committed:
chore(deps): update dependency @nuxtjs/i18n to v8.5.6

renovate[bot] committed:
Merge pull request #6261 from ecamp/renovate/nuxtjs-i18n-8.x

chore(deps): update dependency @nuxtjs/i18n to v8.5.6

renovate[bot] committed:
chore(deps): update dependency @types/node to v22.8.7

renovate[bot] committed:
Merge pull request #6265 from ecamp/renovate/node-22.x

chore(deps): update dependency @types/node to v22.8.7

pmattmann committed:
cleanup

pmattmann committed:
fix: calc absolut element position

renovate[bot] committed:
chore(deps): update dependency globals to v15.12.0

renovate[bot] committed:
fix(deps): update dependency puppeteer-core to v23.7.0

renovate[bot] committed:
chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.13.0

renovate[bot] committed:
Merge pull request #6267 from ecamp/renovate/globals-15.x

chore(deps): update dependency globals to v15.12.0

renovate[bot] committed:
chore(deps): update dependency nuxt to v3.14.0

pmattmann committed:
Merge pull request #6268 from ecamp/renovate/puppeteer-core-23.x

fix(deps): update dependency puppeteer-core to v23.7.0

renovate[bot] committed:
chore(deps): update dependency rector/rector to v1.2.9

renovate[bot] committed:
Merge pull request #6270 from ecamp/renovate/nuxtjs-monorepo

chore(deps): update dependency nuxt to v3.14.0

renovate[bot] committed:
Merge pull request #6269 from ecamp/renovate/typescript-eslint-monorepo

chore(deps): update dependency @typescript-eslint/eslint-plugin to v8.13.0

renovate[bot] committed:
Merge pull request #6271 from ecamp/renovate/rector-rector-1.x

chore(deps): update dependency rector/rector to v1.2.9

renovate[bot] committed:
fix(deps): update dependency phpdocumentor/reflection-docblock to v5.5.0

renovate[bot] committed:
chore(deps): update dependency @types/node to v22.9.0

renovate[bot] committed:
Merge pull request #6273 from ecamp/renovate/phpdocumentor-reflection-docblock-5.x

fix(deps): update dependency phpdocumentor/reflection-docblock to v5.5.0

renovate[bot] committed:
Merge pull request #6274 from ecamp/renovate/node-22.x

chore(deps): update dependency @types/node to v22.9.0

renovate[bot] committed:
fix(deps): update sentry-javascript monorepo to v8.37.1

manuelmeister committed:
Merge pull request #6145 from pmattmann/feature/dashboard-scroll-today

dashboard add scroll-to-today

manuelmeister committed:
Merge pull request #6187 from BacLuc/frontend-fix-jest-is-not-defined

frontend: fix jest is not defined

renovate[bot] committed:
Merge pull request #6276 from ecamp/renovate/sentry-javascript-monorepo

fix(deps): update sentry-javascript monorepo to v8.37.1

usu committed:
make featureToggle.checklist controllable via environment variable

pmattmann committed:
Merge pull request #6277 from usu/feat/checklist-feature-toggle

feat: make featureToggle.checklist controllable via environment variable

pmattmann commented:
/fast-forward
